tests/test_velocity_space.py

```diff
@@ -0,0 +1,169 @@
+"""
```

Comment:
These tests are nearly identical to the auto-lambda tests except with everything done in velocity space.